bin/score.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+
+import json
+import sys
+
+
+def score_submission(predictions_path: str, status: str) -> dict:
+    """Determine the score of a submission. This is a placeholder function.
+
+    Args:
+        predictions_path (str): path to the predictions file
+        status (str): current submission status
+
+    Returns:
+        result (dict): dictionary containing score, status and errors
+    """
+    if status == "INVALID":
+        score_status = "INVALID"
+        score = None
+    else:
+        # placeholder file reading
+        with open(predictions_path, "r") as sub_file:
+            predictions_contents = sub_file.read()
+        try:
+            # placeholder scoring
+            score = 1 + 1
+            score_status = "SCORED"
+            message = ""
+        except Exception as e:
+            message = f"Error {e} occurred while scoring"
+            score = None
+            score_status = "INVALID"
+    result = {"score": {"auc": score, "status": score_status, "errors": message}}
```

Comment:
Nit: In the future, if we wanted to provide more structure and have a "score" object, we may want to have the structure be something like:

```
{"score": {"score_category": "auc", "score": ...., "status"....})
```

Reply:
I added this additional structure to allow for other "score_categories"